packages/react-router/__tests__/data-memory-router-test.tsx

```diff
@@ -2104,19 +2120,23 @@ describe("<DataMemoryRouter>", () => {
         return <p>Await Error:{data.message}</p>;
       }
 
-      return { container, barDefer };
+      return {
+        container: container.querySelector("#content"),
+        barDefer,
+        bazDefer,
+        getAwaitRenderCount() {
+          return awaitRenderCount;
+        },
+      };
     }
 
     it("allows loaders to returned deferred data (child component)", async () => {
       let { barDefer, container } = setupDeferredTest();
       fireEvent.click(screen.getByText("Link to Bar"));
       expect(getHtml(container)).toMatchInlineSnapshot(`
-        "<div>
-          <a
-            href=\\"/bar\\"
-          >
-            Link to Bar
-          </a>
```

Comment:
Shrunk the area of the DOM we were asserting in existing tests since I added a new link to `/baz`